src/libpcp_web/src/query_parser.y

```diff
@@ -960,7 +981,8 @@ static const struct {
 } func[] = {
     { L_AVG,		sizeof("avg")-1,	"avg" },
     { L_COUNT,		sizeof("count")-1,	"count" },
-    { L_MAX,		sizeof("max")-1,	"max" },
```

Comment:
I think we should keep "max" since its part of the language now and people are using it (so if we remove it, things will break) - we can make it a synonym of "max_inst" though.